### PR TITLE
Mark unpartitioned table in HiveWrittenPartitions

### PR DESCRIPTION
**Pull request by shixuan-fan**
We already use "\<UNPARTITIONED\>" for unpartitioned table input, and
this commit would unify the representation of unpartitioned table in
output as well.

### CODE REVIEW

**Comment by charnahalau** [comment]
Thank you. Current change will not brake empty partition case?

**Comment by shixuan-fan** [comment]
@charnahalau I think they should not break empty partition case and only affect unpartitioned table.

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java [hunk 1]
```diff
@@ -1489,6 +1492,7 @@ else if (partitionUpdate.getUpdateMode() == NEW || partitionUpdate.getUpdateMode
         return Optional.of(new HiveWrittenPartitions(
                 partitionUpdates.stream()
                         .map(PartitionUpdate::getName)
+                        .map(name -> name.isEmpty() ? UNPARTITIONED_ID : name)
```
[the remark this answers is not in the document]
Comment (wenleix) [reply presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java]: @raghavsethi  This will be the OperatorInfo in OperatorStats. So it's our internal convention about how to representation about how to represent unpartitioned table. Using a dedicated string instead of empty string should help downstream consumer :)

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java [hunk 2]
```diff
@@ -1210,12 +1214,11 @@ public HiveOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
                         update.getWritePath(),
                         partitionStatistics);
             }
+            return Optional.of(new HiveWrittenPartitions(
```
Comment (wenleix) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java]: This looks unrelated change. But doesn't hurt :)
